feature-libs/customer-ticketing/components/customer-ticketing.service.ts

```diff
@@ -1,14 +1,16 @@
 import { Injectable } from '@angular/core';
```

Comment:
let's delete this file actually since this will not exist anyhow in the future for any upcoming PRs.

Just move the mocked data here https://github.com/SAP/spartacus/pull/16085/files#r968614886

Don't worry about injecting a facade as this is only for the UI anyhow

Reply:
It will require changes in other components and unit tests as well, I already removed this in other PR. so I think we can keep this one for now. I am moving all logic related to messaging in the component to save future conflicts.